### ISSUE TITLE
DSDT namespace problem leads to standby issues on Dell Precision 3561

### ISSUE THREAD

**Issue by watologo1** [acpica/acpica#828]
When trying to enter suspend to ram/standby mode, I see the messages pasted in the end on a black vga screen.
screen goes black, but powersavings are more or less non-existing. Laptop's battery ends after the one or other hour, similar as if it would just run.

Not sure, but it looks as if the bit field variable D6F0 is in scope \PC00 but it gets used in _L69 which seem to be in _GPE scope?

I have sporadically seen kernel graphics driver warnings which may also be related:
i915 0000:00:02.0: drm_WARN_ON(dig_port->tc_lock_wakeref)
915 0000:00:02.0: drm_WARN_ON(dig_port->tc_mode != TC_PORT_DISCONNECTED)

Find acpidump, kernel acipi errors when triggering standby and graphics card kernel warnings:
[acpi_data.tar.gz](https://github.com/acpica/acpica/files/10473311/acpi_data.tar.gz)

ACPI Error: AE_NOT_FOUND, while evaluating GPE method [_L69] (20220331/evgpe-511)
ACPI Error: Aborting method \_GPE._L69 due to previous error (AE_NOT_FOUND) (20220331/psparse-529)
ACPI BIOS Error (bug): Could not resolve symbol [\_GPE._L69.D6F0], AE_NOT_FOUND (20220331/psargs-330)

[drm:nv_drm_gem_fence_attach_ioctl [nvidia_drm]] *ERROR* [nvidia-drm] [GPU ID 0x00000100] Failed to lookup gem object for fence attach: 0x0000000a
ACPI Error: Aborting method \_SB.PC00.LPCB.ECDV._Q66 due to previous error (AE_AML_NOT_OWNER) (20220331/psparse-529)
ACPI Error: Thread 3780640768 cannot release Mutex [ECMX] acquired by thread 3314253824 (20220331/exmutex-378)

Thanks in advance for any help.

**Comment by Wer-Wolf** [comment]
Did you try updating your BIOS?

**Comment by watologo1** [comment]
Yep, I did that several times already. It should not be older than some weeks.

**Comment by Wer-Wolf** [comment]
The latest version is 1.19.0, just in case. Are you running a dualboot setup? If yes, then maybe you could check if the same problem happens in Windows. Because if this is the case, then its likely a BIOS bug, which should be fixed by Dell. In such a case you could talk to Dell support. If the same problem does not happen in Windows, then the problem might be indeed a bug in ACPICA. Can you test if the problem also happens if the proprietary nvidia driver is not loaded?

You can aslo submit a bugreport on the bugtracker of your distribution regarding the errors of the i915 driver.
